guides/v15-mdc-migration.md

```diff
@@ -491,47 +498,47 @@ TODO(mmalerba): link to density docs once they exist.
   is the same width as the form-field
 
 ### Slide Toggle
+<!-- TODO(zarend): review -->
 
-* Accessibility: MDC-based version uses `<button role="switch">` to represent the toggle rather than
-  `<input type="checkbox">`
+* To improve accessibility, the MDC-based version uses `<button role="switch">` to represent the
+  toggle rather than `<input type="checkbox">`.
 
-* Accessibility: The touch target is much larger than the slide toggle; to match the legacy size,
-  you can provide a `-5` density to the slide toggle’s theme mixin:
+* The touch target is much larger and more accessible. Be sure to allow enough space in your
+  layout so that the touch target does not overlap other components. If you are not concerned with
+  accessibility you can match the previous size by using density -5 for the slide-toggle.
 
   ```scss
   @use '@angular/material' as mat;
-
-  @include mat.slide-toggle-theme(
-    map-merge($theme, (density: -5))
-  );
+  @include mat.slide-toggle-density(-5px);
   ```
 
 * The label is closer to the enabled toggle
 
 ### Slider
+<!-- TODO(wagnermaciel): review -->
```

Comment:
@wagnermaciel I know this section still needs an update, I'm just going to leave a TODO here as a reminder